### ISSUE TITLE
LaTeX normalization

### ISSUE THREAD

**Issue by with-him777** [felix-schmitt/MathNet#2]
Hello, I would like to know about the LaTeX normalization processing part. Upon reviewing the training code, it seems that only preprocessing/improve_tokens.py is used for all the processing. Is all the handling done only in improve_tokens.py, or what are the other Python files in the preprocessing folder used for?

**Comment by felix-schmitt** [comment]
Yes, for the normalization you only need the preprocessing/improve_tokens.py file. The other files are for tokenize latex equations or converting datasets in the format used by im2latexv2.
